update_version.sh

```diff
@@ -56,15 +57,15 @@ generateJson() {
   local name="${RELEASE_VERSION}"
   local version="${BUILD_SOURCEVERSION}"
   local productVersion="${RELEASE_VERSION}"
-  local timestamp=$(node -e 'console.log(Date.now())')
+  timestamp=$(node -e 'console.log(Date.now())'); local timestamp
```

Comment:
please declare the variable local before using them

Reply:
Effectively there's no difference here, however, I again was unsure whether the previous one liner was a style preference. I believe this is the most correct:
```bash 
local sha1hash
sha1hash=$(awk '{ print $1 }' "assets/${ASSET_NAME}.sha1")
local sha256hash
sha256hash=$(awk '{ print $1 }' "assets/${ASSET_NAME}.sha256")
```

Reply:
maybe `local sha1hash sha256hash`

But based, on SC2155, even `local name="${RELEASE_VERSION}"` would need to be splitted.

Maybe declaring all the local variables at the beginning of the function would be better?

Reply:
I agree on both counts. Declaring `local`, `readonly`, etc at the top of the script can be a lot clearer.